### ISSUE TITLE
noVNC unable to connect console while WEB browser language not English

### ISSUE THREAD

**Issue by leolleeooleo** [apache/cloudstack#4940]
##### ISSUE TYPE
 * Improvement Request

##### COMPONENT NAME
~~~
noVNC consoleproxy System VMs
~~~

##### CLOUDSTACK VERSION
~~~
4.15
~~~

##### OS / ENVIRONMENT
CentOS 7.9

##### SUMMARY
noVNC unable to connect console while WEB browser language not English

##### STEPS TO REPRODUCE
Here I found 2 ways to make noVNC works fine

1. Set English to first language in the WEB browser (Chrome).
~~~
Setting > Language > Add Language > Set English to first.
~~~

2. Force noVNC language in English
~~~
1. connect to consoleproxy VM 
    sudo ssh -i /root/.ssh/id_rsa.cloud -p 3922 root@169.254.244.225
2. edit javascript
    vi /usr/local/cloud/systemvm/noVNC/app/localization.js
3. At about line 23:
    setup(supportedLanguages) {
        this.language = 'en'; // Default: US English
        return;         //   <<<<<<<<<<<<<  add this line
~~~

And you need to clear cache on your browser.

**Comment by rhtyd** [comment]
@leolleeooleo I think this has been fixed in latest 4.15 branch, can you test latest 4.15 branch? cc @davidjumani @weizhouapache

**Comment by leolleeooleo** [comment]
Got it.
https://github.com/apache/cloudstack/pull/4833
